whois-api/src/main/java/net/ripe/db/whois/api/rest/WhoisCrossOriginFilter.java

```diff
@@ -0,0 +1,74 @@
+package net.ripe.db.whois.api.rest;
+
+import com.google.common.net.HttpHeaders;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletRequestWrapper;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.ws.rs.HttpMethod;
+import jakarta.ws.rs.core.UriBuilder;
+import org.apache.commons.lang.StringUtils;
+import org.eclipse.jetty.http.HttpStatus;
+import org.eclipse.jetty.servlets.CrossOriginFilter;
+
+import java.io.IOException;
+
+public class WhoisCrossOriginFilter extends CrossOriginFilter {
+
+    @Override
+    public void doFilter(final ServletRequest request, final ServletResponse response, final FilterChain chain) throws IOException, ServletException {
+
+        final HttpServletRequest httpRequest = (HttpServletRequest) request;
+
+        // GET request does not trigger a pre-flight request
+        if(httpRequest.getMethod().equals(HttpMethod.GET)
+                && httpRequest.getPathInfo().contains("syncupdates")
```

Comment:
Why hardcode "syncupdates" ?

Reply:
Because we allow updates using get only in case of syncupdates and the comment above mentiosn that since get does not trigger pre-flight request like POST and PUT , request in backend will still be successful but browser will show cross origin failed but till by then object has been updated using GET request

Reply:
OK, no CORS requests allowed to syncupdates API.